### PR TITLE
doc: react doc link

### PR DESCRIPTION
**Pull request by bbcvc**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |     e6ec1c03f569661c06b86784a0996cef1ca134ac      |

### CODE REVIEW

File: components/transfer/index.zh-CN.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Transfer 支持接收 `children` 自定义渲染列表，并返回以下参数
 
 ## 注意
 
-按照 React 的[规范](http://facebook.github.io/react/docs/lists-and-keys.html#keys)，所有的组件数组必须绑定 key。在 Transfer 中，`dataSource` 里的数据值需要指定 `key` 值。对于 `dataSource` 默认将每列数据的 `key` 属性作为唯一的标识。
+按照 React 的[规范](https://react.dev/learn/rendering-lists#why-does-react-need-keys)，所有的组件数组必须绑定 key。在 Transfer 中，`dataSource` 里的数据值需要指定 `key` 值。对于 `dataSource` 默认将每列数据的 `key` 属性作为唯一的标识。
```
Comment (li-jia-nan) [inline components/transfer/index.zh-CN.md:90]: 英文文档也顺便改一下？

Comment (li-jia-nan) [inline components/transfer/index.zh-CN.md:90]:
```suggestion
按照 React 的[规范](https://zh-hans.react.dev/learn/rendering-lists#why-does-react-need-keys)，所有的组件数组必须绑定 key。在 Transfer 中，`dataSource` 里的数据值需要指定 `key` 值。对于 `dataSource` 默认将每列数据的 `key` 属性作为唯一的标识。
```

Comment (bbcvc) [reply components/transfer/index.zh-CN.md:90]: 好，我提个新的pr吧，这个在网页上改的有的不太方便修改

Comment (bbcvc) [reply components/transfer/index.zh-CN.md:90]: https://github.com/ant-design/ant-design/pull/49766

**Pull request closed without merging**